src/components/mint-button-primary/mint-button-primary.scss

```diff
@@ -0,0 +1,312 @@
+@import '~sass/main.scss';
```

Comment:
This scss should be generalized for 2 components. 
Right now the file is copied twice.
TBD